src/libraries/System.ComponentModel.Primitives/tests/System/ComponentModel/ComponentTests.cs

```diff
@@ -219,6 +219,7 @@ public void Dispose_InvokeDisposingCannotRaiseEvents_DoesNotCallHandler(bool dis
         }
 
         [Fact]
+        [ActiveIssue("https://github.com/dotnet/runtime/issues/72494", typeof(PlatformDetection), nameof(PlatformDetection.IsNativeAot))]
```

Comment:
Can you make this a ConditionalFact and condition it on a new method ("IsInvokingFinalizersSupported"?) similar to the existing:

https://github.com/dotnet/runtime/blob/a54a823ece1094dd05b7380614bd43566834a8f9/src/libraries/Common/tests/TestUtilities/System/PlatformDetection.cs#L141

This was a conscious design choice. Nothing fundamentally preventing doing this, but it's work, and there's very little legitimate scenarios for doing this (nobody ever ran into this in .NET Native).